.gitignore

```diff
@@ -26,12 +26,15 @@ src/modules.order
 /configs/*/emc.nml
 !/configs/common/emc.nml
 /docs/src/source-highlight/local
+docs/src/*/gui/gmoccapy_release_notes.txt
```

Comment:
Why?

Reply:
They are pre existing, I just moved them near more aligned rules as they looked quite silly following the commented line `# Ignore VSCode settings`